src/sql/src/plan/statement/ddl.rs

```diff
@@ -2814,6 +2815,49 @@ pub fn plan_create_secret(
     }))
 }
 
+pub fn describe_create_connector<T: mz_sql_parser::ast::AstInfo>(
+    _: &StatementContext,
+    _: &CreateConnectorStatement<T>,
+) -> Result<StatementDesc, anyhow::Error> {
+    Ok(StatementDesc::new(None))
+}
+
+pub fn plan_create_connector(
+    scx: &StatementContext,
+    stmt: CreateConnectorStatement<Aug>,
+) -> Result<Plan, anyhow::Error> {
+    scx.require_experimental_mode("CREATE CONNECTOR")?;
+
+    let create_sql = normalize::create_statement(&scx, Statement::CreateConnector(stmt.clone()))?;
+    let CreateConnectorStatement {
+        name,
+        connector,
+        if_not_exists,
+    } = stmt;
+    let connector_literal = match connector {
```

Comment:
nit: this var is `connector_literal`, but think it wants to be `connector_inner`--not worth going back through CI for, though